web/src/invite.ts

```diff
@@ -39,6 +39,10 @@ let custom_expiration_time_unit = "days";
 let pills: email_pill.EmailPillWidget;
 let stream_pill_widget: stream_pill.StreamPillWidget;
 
+type PillContainerJQuery = {
+    data: (key: "pill_widget") => stream_pill.StreamPillWidget | undefined;
```

Comment:
I would suggest creating a `clear_stream_pills` function within the `invite_stream_picker_pill.ts` file. However, I'm not entirely sure of any other best approach at the moment.